### ISSUE TITLE
Query "too long to run" when an offset is added.

### ISSUE THREAD

**Issue by Llamatic** [datagovuk/apiserver#79]
Even know I set a limit of 4999 it still says "Query would take too long to run, please apply a LIMIT of less than 5000 rows"

This is the SQL query I did on the Health SQL page:
SELECT * FROM social_care_locations OFFSET 14996 LIMIT 4999

It seems that if an OFFSET is above a certain number it would show the "Query would take too long to run" error.

**Comment by rossjones** [comment]
Yeah, it's not a perfect solution, there's also the 5s timeout which is probably catching you in this case. As it was built as a proof of concept, I doubt very much this'll get fixed though.

If you're just trying to grab the entire dataset, perhaps it would be easier to just download the source file from https://data.gov.uk/dataset/social-care-locations ?
